in-toto-spec.md

```diff
@@ -266,69 +269,84 @@ systems that further verify the integrity of each step.
 Enhancements to the in-toto specification can be submitted as
 [ITEs](https://github.com/in-toto/ITE).
 [ITE-1](https://github.com/in-toto/ITE/blob/master/ITE/1/README.adoc) describes
-this process in greater detail.
+this process in greater detail. After an ITE is accepted, the changes it
+describes may be integrated into this document. Each version of the
+specification indicates the ITEs it adheres to.
```

Comment:
Does it make sense to say that the spec "adheres" to a non-"standards" ITE? ITE-1 is of type "process" and ITE-2 and ITE-3 are "informational", they don't seem to affect the specification. Should we clarify that in some way, or just not mention them?

Reply:
Perhaps we can find an alternative to "adhere". I think we should list them because we call back to them for layout distribution in a couple of places including sec 5.2.

cc @jhdalek55

Reply:
I separated out the informational ones and used "recommends".

```
Finally, this version of the specification is aware of and recommends the use of
semantics described in the following ITEs:
* [ITE-2](https://github.com/in-toto/ITE/blob/master/ITE/2/README.adoc)
* [ITE-3](https://github.com/in-toto/ITE/blob/master/ITE/3/README.adoc)

ITE-2 and ITE-3 describe semantics for bootstrapping trust for in-toto layouts
and their public keys. The same mechanisms are used to associate link metadata
files with each artifact being verified.
```